sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -90,9 +89,12 @@ def _generate_file_prefix(pipeline_gcs_location):
 
 
 def _make_new_file_writer(file_prefix, destination):
-  if isinstance(destination, bigquery_api.TableReference):
-    destination = '%s:%s.%s' % (
-        destination.projectId, destination.datasetId, destination.tableId)
+  destination = bigquery_tools.get_hashable_destination(destination)
```

Comment:
Is there a reason to use the `:`? For compatibility with legacy SQL?
https://cloud.google.com/bigquery/docs/reference/standard-sql/migrating-from-legacy-sql#project-qualified_table_names